public-api/src/main/resources/public-api.yaml

```diff
@@ -204,7 +204,7 @@ paths:
       tags:
         - dataBrowser
 
-      description: Gets domain filters
+      description: Gets domain filters (not used?)
```

Comment:
So domain filters were the list of domains we would make check boxes to filter our search down to specific domains.  And in quick-search page we show only those with db_type domain_filter. We also had vocabulary filters at one point.  

Mark as todo and i will take a look. The idea was we just don't do any summing. I don't think it hurts to have it in there. If we cache the counts in db_domain at count generation time, then this can be used. .

Reply:
Will mark as TODO. If the code's not in use, we should consider getting rid of it.

Reply:
Will do in another pr.